services/sso/.eslintrc

```diff
@@ -0,0 +1,19 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "sourceType": "module"
+  },
+  "extends": ["airbnb-base", "prettier"],
+  "plugins": ["jest", "prettier"],
+  "root": true,
+  "ignorePatterns": ["node_modules/**", "src/public/javascript/**", "dist/**", "webpack.**"],
```

Comment:
Doesn't look like a dist dir is being created or webpack is being used so do we need to include those here?

Reply:
This is just copying and pasting from the template. I think it's probably not a good use of time editing these templates to fit every common service. Happy to discuss.

Reply:
Edited.